templates/aws/support.html

```diff
@@ -1,144 +1,158 @@
+        </p>
+      </div>
+      <div class="col p-image-container is-highlighted u-hide--small">
+        {{ image (
+        url="https://assets.ubuntu.com/v1/7d50bc74-ubuntu_pro-aws.svg",
         alt="",
-        width="167",
-        height="100",
+        width="230",
+        height="104",
         hi_def=True,
         loading="auto"
         ) | safe
-      }}
+        }}
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-strip is-bordered">
-  <div class="row">
-    <div class="col-6">
-      <h2>Ubuntu Pro</h2>
-      <p>Ubuntu Pro for AWS is a premium image delivering the most comprehensive open source security and AWS compliance. Ubuntu Pro is suitable for small to large-scale Linux enterprise operations offering pay-as-you-go billing on your existing AWS invoice.</p>
-      <p>For more information about Ubuntu Pro on AWS, see <a href="/aws/pro">https://ubuntu.com/aws/pro</a>.</p>
-    </div>
-    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/d649019e-u+pro+logo+158x40.png",
-        alt="",
-        width="395",
-        height="100",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+  <section class="p-strip">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Ubuntu Pro</h2>
+      </div>
+      <div class="col">
+        <p>
+          Ubuntu Pro for AWS is a premium image delivering the most comprehensive open source security and AWS compliance. Ubuntu Pro is suitable for small to large-scale Linux enterprise operations offering pay-as-you-go billing on your existing AWS invoice.
+        </p>
+        <p>
+          For more information about Ubuntu Pro on AWS, see <a href="/aws/pro">https://ubuntu.com/aws/pro</a>.
+        </p>
+      </div>
     </div>
-  </div>
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-  <div class="row">
-    <div class="col-6">
-      <h2>Ubuntu Pro with 24x7 Support</h2>
-      <p>With 24x7 SLAed support and the widest open source security patching, teams can build on Ubuntu Pro with Support to secure their application's open source supply chain and receive expert assistance when they need it.</p>
-      <p>24x7 Support is available for the Ubuntu operating system (infrastructure) and can be extended to include application support for <a href="/support#apps-support">key open source applications</a>.</p>
-      <p>See more details at <a href="/support">https://ubuntu.com/support</a>.</p>
-    </div>
-    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/e9d3cf66-We+support+your+operations.svg",
-        alt="",
-        width="150",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+  </section>
+
+  <section class="p-strip u-no-padding--top">
```

Comment:
```suggestion
  <section class="p-section">
```
accomplishes the same thing